### PR TITLE
QA Updates

### PR DESCRIPTION
**Pull request by bhenndricks**
# WMS 11440

# General requirements
1. Information in the workshop is adequate and updated
2. Code is correct and working
3. Links are correct
4. Make sure you enter the help email link in your manifest.json
5. Please make sure WMS URLs are updated as needed after your PR is approved

# Checklist - Refer to the QA document for the complete list 
Please confirm that the following is completed before submitting your PR

- [ ] All filenames are lower case (including folders, images, files, etc.)
- [ ] Filenames are descriptive
- [ ] Your workshop folder structure should be similar to the one used in the sample workshop (https://github.com/oracle-livelabs/common/tree/main/sample-livelabs-templates/sample-workshop)
- [ ] Are you using multiple versions (desktop/, sandbox/, tenancy/)? Make sure that each of them contains a manifest.json and an index.html
- [ ] Image references in markdown contain an alternate text

### CODE REVIEW

**Review by kaylien — APPROVED** [review]

File: 23cfree/schema-priv-setup/setup.md [hunk 1]
```diff
@@ -20,105 +20,107 @@ In order to do this workshop you need
 * Insert data into the inventory\_no\_reservations table with two rows: Product A and Product B.
 * Insert data into the inventory\_reservations table with two rows: Product C and Product D.
 
-## 
-## Task 1: Create Users 
+##
```
Comment (anooshapilli) [inline 23cfree/schema-priv-setup/setup.md:23]: @bhenndricks - Could you remove the two ## before Task 1 in all three markdown files?

**Review by anooshapilli — COMMENTED** [review]
@bhenndricks - Please make the requested change and update the workshop status to Self QA Complete in WMS to approve the PR.

**Review by kaylien — APPROVED** [review]

File: 23cfree/schema-priv-setup/setup.md (hunk 1 above)
Comment (kaylien) [reply 23cfree/schema-priv-setup/setup.md:23]: Hi Anoosha,

It's odd but I have gotten on a call with him to look at his files. For some reason if he doesn't have that two ## before Task 1, task 1 will not be underneath the fold. I didn't know how else to resolve it. Perhaps you can help guide him if there's a better way. 

Here's the lab as is: https://oracle-livelabs.github.io/database/23cfree/workshops/sandbox-schemapriv-lockfree/index.html?lab=setup

Comment (kaylien) [reply 23cfree/schema-priv-setup/setup.md:23]: Thanks Anoosha! 
@bhenndricks please create a new commit for this before we can approve your PR

**Comment by bhenndricks** [comment]
Hello, I have implemented the introduction.

**Review by bhenndricks — COMMENTED** [review]
Intro attached

**Review by anooshapilli — APPROVED** [review]

**Pull request merged**